src/Data/Undefined/NoProblem.js

```diff
@@ -1,3 +1,4 @@
-"use strict"
-
-exports.undefined = undefined;
+const undefinedImpl = function () {
+    return undefined
+}
```

Comment:
Hey @sigma-andex, is this change on purpose or an oversight? If on purpose, what was the rationale?

```suggestion
const undefinedImpl = undefined
```

Reply:
@fsoikin sorry I oversaw this comment. yes I guess an oversight

Reply:
Great 👍 
Just wanted to validate my mental model.